packages/react-server/src/ReactFlightReplyServer.js

```diff
@@ -396,10 +396,6 @@ function parseModelString(
         const chunk = getChunk(response, id);
         return chunk;
       }
-      case 'S': {
-        // Symbol
-        return Symbol.for(value.slice(2));
-      }
```

Comment:
This is the functional change since it doesn't matter what the client does if you can still fake a payload that includes this.